md/execution-sequence-states-and-transactions.md

```diff
@@ -4,127 +4,137 @@ This page describes the sequence of actions performed when a process instance is
+## Instantiation of a process through a user entry form
 
-### Initialization
-
-1. The Portal displays the form to the user. 
-2. The user enters data in the form. 
-3. The button of the form having the **Start process** action in the **UI designer** send a POST request with the form data to start the process. 
-4. The submitted data are validated against the **process instantiation contract**.
-5. If the contract is valid the process instance is created and input values are persisted in database.
+1. The Portal displays the process instantiation form to the user
+1. The user enters data in the form
+1. The button of the form having the **Start process** action in the **UI designer** sends a POST request with the form data to start the process
+1. The submitted data are validated against the **process instantiation contract**
+1. If the contract is valid the process instance is created and input values are persisted in database
 6. Process data instances are created and initialized in the order of declaration using
   * An **operation** declared on the process that saves its value to the process data
   * The **initial value expression** if no operation is declared
   * **null** if no expression or operation is defined
-7. **SP only** The Engine creates and initializes search indexes.
-8. The Engine creates and initializes business data.
-9. The Engine initializes process instance Documents with default value coming from the definition or with submitted files.
-10. Operations (that does not initialize data) are executed.
-11. Completing the instantiation The Engine instantiates and executes the "on enter" connectors (evaluate input expression, execute, evaluate output operation). This an asynchronous operation. Execution duration of a connector is limited to a maximum of 5 minutes by default. 
-12. The process is instantiated, the API call finish and the Engine executes the process flow asynchronously.
+7. The Engine creates and initializes business data
+1. **Subscription only**: The Engine creates and initializes search indexes
+1. The Engine initializes process instance Documents with default value coming from the definition or with submitted files
+1. Operations (that does not initialize data) are executed. Operations at process instantiation level can either come from V6 forms or from calling process instances (for processes stared by call-activities)
+1. The Engine instantiates and executes the "on enter" connectors (evaluate input expression, execute, evaluate output operation). Connectors are executed in the order of declaration  
+   **Community**:    Execution duration is not limited. If the connector never finishes, the process instantiation will never finish  
+   **Subscription**: Execution duration of a connector is limited to a maximum of 5 minutes by default. This timeout value can be changed by configuration. Above this limit,
+   the connector execution is aborted and the process instantiation is considered failed.
```

Comment:
The linebreak is here for lisibility when editing the code.
Having lines of 2000 characters is counter-productive.